metrics.md

```diff
@@ -572,6 +572,57 @@ Choice elements:
 Notes:
 - These are the choice fields in US Core, but could use the FHIR definitions to create a full list, or could do a more general query based on JSON key prefix (not supported by all databases and potentially slow even where supported)
 
+### c_attachment_count
+**[structure]** Survey of attachment data and formats
+
+Stratified
+by [status](#by-status),
+by content type,
+by language,
+by presence of data and/or URL fields,
+by format (when applicable).
+
+| Resource Type     | Element       |
+|-------------------|---------------|
+| DiagnosticReport  | presentedForm |
+| DocumentReference | content       |
+
+Notes:
+- This metric counts _attachments_ inside resources and their associated metadata.
+  Most other metrics count resources, but this one is looking at each attachment.
+  Only the status field is taken from the outer resource.
+- When examining the content type, you should strip off extra info like charset
+  information (e.g. `text/plain; charset=utf8`) since the primary focus here is on
+  available MIME types.
+- Only DocumentReference has a `format` field associated with the attachment.
+
+### c_content_type_use
+**[structure]** Distribution of content types among attachment resources
+
+Stratified
+by [status](#by-status),
+by content types,
+by document type,
+by document status (when applicable).
+
+| Resource Type     | Attachment Element | Document Type Element |
+|-------------------|--------------------|-----------------------|
+| DiagnosticReport  | presentedForm      | code                  |
+| DocumentReference | content            | type                  |
+
+Notes:
+- Present all the content types available for a given resource.
+  That is, if a resource has both text/plain and text/html content types, present both.
```

Comment:
Should we include guidance on sorting the mime types being concatenated so they'll be easily comparable between implementations (e.g., alpha desc)?

Reply:
In my head, that level of comparison between two implementations was not a goal. Like, qualifier doesn't specify field names, table names, or other "presentation" kind of questions like "is this data split across multiple tables or not".

I guess those are all structural issues. And "how this string value should be sorted" is more of a data question.

But the current `data-metrics` approach (concatenate in alphabetical order, separated by semicolons) is more of an artifact of how Cumulus Library normally does things, than like, an obvious Best Way approach. Like, why semicolons? (because we often convey data in CSV files and commas would be annoying, plus commas might already be in display strings) So I feel a little bad freezing a Cumulus-ism in place just because I'm writing the description.

Reply:
Yeah, I see this as different than standardizing field names since these values would be surfaced in a dashboard that is being used to review the data present at a site. 

I agree about specifying delimiters though, especially since a system like duckdb could return an inline list if the view layer can handle it instead of concatenating. What if we just specify that when there are multiple mime types returned, they should be ordered alphabetically? Obviously, the view layer could then re-order randomly, but at least this sets an expectation of consistency.

Reply:
Yeah :+1: (they're going to have to sort in some fashion anyway, might as well guide em)

Reply:
Done.